files/galaxy/tpv/tools.yml

```diff
@@ -1085,6 +1085,15 @@ tools:
   toolshed.g2.bx.psu.edu/repos/genouest/braker3/braker3/3.0.7.*:
     mem: 4
     cores: 6
+    params:
+      singularity_container_id_override: "docker://teambraker/braker3:v3.0.7.1"
+    scheduling:
+      require:
+        - singularity
+
+  toolshed.g2.bx.psu.edu/repos/genouest/braker3/braker3/3.0.8.*:
```

Comment:
this 3.0.8 version would now use a 3.0.7.3 docker container, is that correct?

Reply:
I updated it this morning via this [PR](https://github.com/usegalaxy-eu/infrastructure-playbook/pull/1107). I was unaware that a new version of the tool would be released.

Reply:
I will install the new version atm. @sanjaysrikakulam can you merge if you think its ready?

Reply:
Merged!